src/main/java/com/scrive/pdftools/tools/flattener/FlattenerSpec.java

```diff
@@ -0,0 +1,18 @@
+package com.scrive.pdftools.tools.flattener;
+
+import com.scrive.pdftools.tools.utils.FileInput;
+import org.json.JSONObject;
+
+public class FlattenerSpec {
+    public FileInput mainFileInput = null;
```

Comment:
let's not use public fields in Java.

replace them with getters and consider if they couldn't be made even `final` (initialize them in constructor)

Reply:
Changed so they are made in the constructor.